samples/settings/boards/nrf52840dk_nrf52840.conf

```diff
@@ -5,8 +5,6 @@ CONFIG_WIFI_ESP_AT_MDM_RX_BUF_COUNT=40
 CONFIG_NET_L2_WIFI_SHELL=y
 CONFIG_GOLIOTH_SAMPLE_WIFI=y
 
-CONFIG_MCUMGR_SMP_BT=y
-CONFIG_MCUMGR_SMP_BT_AUTHEN=n
 CONFIG_BT_L2CAP_TX_MTU=252
 CONFIG_BT_BUF_ACL_RX_SIZE=256
```

Comment:
Please remove
```
CONFIG_BT_L2CAP_TX_MTU=252
CONFIG_BT_BUF_ACL_RX_SIZE=256
```
as BT is no longer enabled (indirectly by `CONFIG_MCUMGR_SMP_BT=y`). Those probably generate warnings right now at cmake configuration stage.